src/nvim/edit.c

```diff
@@ -609,9 +609,6 @@ static int insert_execute(VimState *state, int key)
   InsertState *s = (InsertState *)state;
   s->c = key;
 
-  // Don't want K_EVENT with cursorhold for the second key, e.g., after CTRL-V.
-  did_cursorhold = true;
```

Comment:
Thanks for your insights and help!
It sounds reasonable, but I cannot really say much about this - except for that removing globals is a good thing always.. ;)